From 902d68a65813ecdd092d553775a40eac243197a4 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 5 Mar 2019 16:17:11 -0800
Subject: [PATCH] TRITON-1286 test-amon failure in nightly-2:
 maintenances.test.js: timeout in "maint 2: stop amontestzone" Reviewed by:
 Chris Burroughs <chris.burroughs@joyent.com> Approved by: Chris Burroughs
 <chris.burroughs@joyent.com>

---
 test/maintenances.test.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/maintenances.test.js b/test/maintenances.test.js
index acc9f45..2403881 100644
--- a/test/maintenances.test.js
+++ b/test/maintenances.test.js
@@ -276,7 +276,7 @@ var maint1AlarmId;
 
 test('maint 1: stop amontestzone', {timeout: TIMEOUT_MAX}, function (t) {
     notifications = []; // reset
-    common.vmStop({uuid: prep.amontestzone.uuid, timeout: 3 * 60 * 1000},
+    common.vmStop({uuid: prep.amontestzone.uuid, timeout: 10 * 60 * 1000},
         function (err) {
             t.ifError(err, 'stopped amontestzone');
             t.end();
@@ -451,7 +451,7 @@ test('maint 2: create maint window', function (t) {
 test('maint 2: stop amontestzone', {timeout: TIMEOUT_MAX}, function (t) {
     notifications = []; // reset
     // I've seen this VM stop on nightly-2 take >1 min.
-    common.vmStop({uuid: prep.amontestzone.uuid, timeout: 3 * 60 * 1000},
+    common.vmStop({uuid: prep.amontestzone.uuid, timeout: 10 * 60 * 1000},
         function (err) {
             t.ifError(err, 'stopped amontestzone');
             t.end();
-- 
2.21.0

